config/data/models.json

```diff
@@ -40,6 +40,12 @@
         "distribution_path": "meta-llama/Meta-Llama-3-70B-Instruct",
         "transformer_params_key": "Meta-Llama-3-70B"
     },
+    "meta-llama/Meta-Llama-3.1-8B-Instruct": {
```

Comment:
nit: technically you can also add support for the pre-trained model ([meta-llama/Meta-Llama-3.1-8B](https://huggingface.co/meta-llama/Meta-Llama-3.1-8B)) and the llama guard ([meta-llama/Llama-Guard-3-8B](https://huggingface.co/meta-llama/Llama-Guard-3-8B)). Not a requirement though

Reply:
Probably want support for the pre-trained (non-Instruct) version to match with our support for Llama 3.0?

Reply:
oh yeah of course. This PR is focused on enabling llama3.1 in torchchat, so didn't cover all possible models. Will have another PR to handle that.